### PR TITLE
Addition of a Gothic Loyalist trait.

### PR DESCRIPTION
**Pull request by CaudilloFranco**
This is added to make the events I made work. I will make a .gfx file edit (in WtWSMS\interface\WtWSMS.gfx or whatever) when possible.

### CODE REVIEW

File: WTWSMS/common/traits/WtWSMS_new_traits_other.txt [hunk 1]
```diff
@@ -59,4 +59,17 @@ legendary_leader = {
 	ai_ambition = 300
 	ai_greed = -300
 	ai_zeal = 10
-}
\ No newline at end of file
+}
+gothic_loyalist = {
+	culture = ostrogothic
```
Comment (CaudilloFranco) [inline WTWSMS/common/traits/WtWSMS_new_traits_other.txt]: Would this work if I wanted only Ostrogoths to get this event or will this break the game?

**Comment by rquinio** [comment]
I think adding new traits is generally not save-compatible due to saves referencing a numeric ID based on load order, instead of the full trait name.
You could try putting it in a file WtWSMS_zz_traits.txt, so that it is loaded last in alphanumeric order, which is how vanilla maintains compatibility.

Even though I'm not sure if master is not already save-incompatible with 0.9.6 due to some removals of culture/religion :)

File: WTWSMS/common/traits/WtWSMS_new_traits_other.txt (hunk 1 above)
Comment (rquinio) [inline WTWSMS/common/traits/WtWSMS_new_traits_other.txt]: It won't work, you need to move the culture check inside potential block

File: WTWSMS/common/traits/WtWSMS_new_traits_other.txt [hunk 2]
```diff
@@ -59,4 +59,17 @@ legendary_leader = {
...
+	potential = {
+		age = 16
+	}
+	learning = 1
+	inherit_chance = 65
+	opposites = { 
+		trait = roman_nostalgia
```
Comment (rquinio) [inline WTWSMS/common/traits/WtWSMS_new_traits_other.txt]: You don't need trait = in opposites block:
opposites =  {
	roman_nostalgia
}

**Comment by loup99** [comment]
I have removed the duplicate branch.

**Comment by CaudilloFranco** [comment]
I made a new pull request but this time in a different file and edited as well.

**Pull request closed without merging**